content/en/getting_started/application/_index.md

```diff
@@ -27,93 +27,145 @@ further_reading:
+Datadog [Real User Monitoring][11] (RUM) allows you to visualize and analyze real-time user activities and experiences. With [Session Replay][12], you can capture and view the web browsing sessions of your users to better understand their behavior.
+
+**Key capabilities**:
+- Visualize load times, frontend errors, and resources for every user session.
+- Filter and analyze data using custom attributes.
+- Unify frontend and backend metrics in one view for faster troubleshooting.
+
+Navigate to the [RUM explorer][13] in the app to get started. To learn more, read the [RUM documentation][11]
 
 ## Synthetic Monitoring
 
-Datadog [Synthetic Monitoring][22] allow you to create and run API and browser tests that proactively simulate user transactions on your applications and monitor all internal and external network endpoints across your system's layers. You can detect errors, identify regressions, and automate rollbacks to prevent issues from surfacing in production.
+Datadog [Synthetic Monitoring][14] allows you to create and run API, browser, and mobile tests that proactively simulate user transactions on your applications. These tests monitor all internal and external network endpoints across your system's layers. 
+
+**Key capabilities**:
+
+- Test business-critical API endpoints and user journeys.
+- Detect errors, identify regressions, and automate rollbacks to prevent issues from surfacing in production.
+- Find and alert on performance issues for users in various locations.
+
+Navigate to [Synthetic Monitoring & Testing][15] in the app to get started. To learn more, read the [Synthetic Monitoring documentation][14].
 
 ## Integrations
 
-{{< img src="getting_started/application/integrations-2024.png" alt="Integrations" >}}
+Use Datadog's {{< translate key="integration_count" >}} [integrations][16] to bring together all of the metrics and logs from your infrastructure and gain insights into your entire observability system.
 
-- Datadog has over {{< translate key="integration_count" >}} integrations [officially listed][2].
-- Custom integrations are available through the [Datadog API][3].
-- The Agent is [open source][4].
+{{< img src="getting_started/application/integrations-2025.png" alt="Integrations" >}}
+
+**Key capabilities**:
+
+- Available integrations cover cloud technologies, incident response, data layers, security, AI, and more.
+- Build your own integration using the [developer documentation][17].
 - After integrations have been configured, all data is treated the same throughout Datadog, whether it is living in a data center or in an online service.
```

Comment:
```suggestion
- After integrations have been configured, all data is treated the same throughout Datadog, whether it is living in a data center or in an online service.
- Build your own integration using the [developer documentation][17].
```
I'd switch these two bullets, cos the I think the user is most likely to be interested in the first two.